examples/upgrade/upgrade-force.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: elemental.cattle.io/v1beta1
+kind: ManagedOSImage
+metadata:
+  name: upgrade3
+  namespace: fleet-default
+spec:
+  clusterTargets:
+    - clusterName: my-cluster
+  upgradeContainer:
+    command:
```

Comment:
This can be simplified based on https://github.com/rancher/elemental-operator/pull/436
You can simply add to default ManagedOSImage created by the UI.
```yaml
upgradeContainer:
  envs:
  - name: FORCE
    value: "true"
```

Reply:
Good point, I thought it would be valuable to show available fields, but might just make it confusing. Will simplify it! :+1: